ui/dashboards/src/context/DashboardProvider/layout-slice.tsx

```diff
@@ -42,16 +36,19 @@ export interface LayoutSlice {
   /**
    * Updates an existing panel group to, for example, change its display properties.
    */
-  updatePanelGroup: (layout: LayoutDefinition, groupIndex?: number) => void;
+  updatePanelGroup: (layout: Omit<GroupDefinition, 'id'>, groupIndex?: number) => void;
 
   /**
    * Rearrange the order of panel groups by swapping the positions
    */
   swapPanelGroups: (xIndex: number, yIndex: number) => void;
 }
 
-export interface PanelGroupDefinition extends LayoutDefinition {
+export interface GroupDefinition {
   id: number;
+  items: GridItemDefinition[];
+  isOpen?: boolean;
```

Comment:
Do you think it would be worth calling this out as the "default" is open state? (e.g. `isOpenDefault` or something)